examples/CONTRIBUTING.md

```diff
@@ -0,0 +1,43 @@
+# Contributing
+
+See [CONTRIBUTING.md](../CONTRIBUTING.md) for the general guides for contribution.
+
+## Keeping create-yorkie-app in sync with examples
+
+When adding a new example, you have to update create-yorkie-app's [FRAMEWORKS.ts](../tools//create-yorkie-app/FRAMEWORKS.ts).
```

Comment:
I think the `FRAMEWORKS.ts' file name shouldn't be capitalized. lowercase would be more conventional.